src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -17,6 +17,87 @@ public class DockerHelper
         private const string dockerPSCommand = "ps";
         // --no-trunc avoids parameter truncation
         private const string dockerPSArgs = "-f status=running --no-trunc --format \"{{json .}}\"";
+        private const string dockerInspectCommand = "inspect";
+        private const string dockerInspectArgs = "-f \"{{json .Platform}}\" ";
+
+        public static string GetContainerPlatform(string hostname, string containerName)
```

Comment:
I would convert this to `bool TryGetContainerPlatform(string hostname, string containerName, out string platform)`